### PR TITLE
PMM-8211 verification of resources and secrets deletion checks dbaas

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

File: tests/DbaaS/verifyDBaaSPXCCluster_test.js [hunk 1]
```diff
@@ -299,3 +299,77 @@ Scenario('PMM-T717 Verify insufficient resources warning @dbaas @nightly',
...
+    I.waitForDetached(dbaasPage.tabs.kubernetesClusterTab.disabledAddButton, 30);
+    await dbaasActionsPage.createClusterAdvancedOption(clusterName, pxc_resource_check_cluster_name, 'MySQL', pxc_configuration);
+    I.click(dbaasPage.tabs.dbClusterTab.createClusterButton);
+    I.waitForText('Processing', 30, dbaasPage.tabs.dbClusterTab.fields.progressBarContent);
+    await dbaasPage.postClusterCreationValidation(pxc_resource_check_cluster_name, clusterName);
+    const {
+      username, password, host, port,
+    } = await dbaasAPI.getDbClusterDetails(pxc_resource_check_cluster_name, clusterName);
+
+    output = await I.verifyCommand(
```
Comment (yurkovychv) [inline tests/DbaaS/verifyDBaaSPXCCluster_test.js:337]: is there a need to assign it to a variable ?

Comment (puneet0191) [reply tests/DbaaS/verifyDBaaSPXCCluster_test.js:337]: no

Comment (puneet0191) [reply tests/DbaaS/verifyDBaaSPXCCluster_test.js:337]: will change it, good point

**Review by beatahandzelova — APPROVED** [review]

File: tests/DbaaS/verifyDBaaSPXCCluster_test.js [hunk 2]
```diff
@@ -299,3 +299,72 @@ Scenario('PMM-T717 Verify insufficient resources warning @dbaas @nightly',
...
+      memory: '1 GB',
+      cpu: '1',
+      disk: '3 GB',
+      dbType: 'MySQL',
+      clusterDashboardRedirectionLink: dbaasPage.clusterDashboardUrls.pxcDashboard(
+        pxc_resource_check_cluster_name,
+      ),
+    };
+
+    if (!await dbaasAPI.apiCheckRegisteredClusterExist(clusterName)) {
```
Comment (yurkovychv) [inline tests/DbaaS/verifyDBaaSPXCCluster_test.js:321]: I think this is done in BeforeEach hook

**Review by yurkovychv — APPROVED** [review]
All good, thanks! were you able to test in on CI?

File: tests/DbaaS/verifyDBaaSPXCCluster_test.js (hunk 2 above)
Comment (puneet0191) [reply tests/DbaaS/verifyDBaaSPXCCluster_test.js:321]: yeah, will change it

**Comment by puneet0191** [comment]
checked two times, job is green

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**